File: despatch.WEB/app/src/swagger/models/contact-data-images-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataRequest } from './contact-data-request';
import { ImageRequest } from './image-request';
export interface ContactDataImagesRequest extends BaseEntityRequest{
  contactData?: ContactDataRequest;
  contactDataId?: string;
  image?: ImageRequest;
  imageId?: string;
}

File: despatch.Domain/Models/Default/Request/ContactDataImagesRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataImagesRequest : BaseEntityRequest
    {
        public string ContactDataId { get; set; }
        public string ImageId { get; set; }

        public virtual ContactDataRequest ContactData { get; set; }
        public virtual ImageRequest Image { get; set; }
    }
}
